ledger/eval/txntracer_test.go

```diff
@@ -206,15 +260,19 @@ int 1`,
 							},
 						},
 					},
+					{
+						Addr:        innerBoxAppAddress,
+						AccountData: ledgercore.AccountData{},
+					},
 				},
 				AppResources: []ledgercore.AppResourceRecord{
 					{
 						Aidx: 1 + offset,
 						Addr: addrs[0],
 						Params: ledgercore.AppParamsDelta{
 							Params: &basics.AppParams{
-								ApprovalProgram:   []byte{0x06, 0x80, 0x05, 0x68, 0x65, 0x6c, 0x6c, 0x6f, 0xb0, 0x81, 0x01},
-								ClearStateProgram: []byte{0x06, 0x81, 0x01},
+								ApprovalProgram:   []byte{0x08, 0x80, 0x08, 0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x62, 0x6f, 0x78, 0x81, 0x0a, 0xb9, 0x48, 0x81, 0x01},
+								ClearStateProgram: []byte{0x08, 0x81, 0x01},
```

Comment:
I guess you didn't put these here originally, but the fact that we're editing hex bytecode here seems bad.  I assume this is a compiled program, can we use `logic.AssembleString` so it makes more sense?

Reply:
Updated. I actually did write those originally, just not in this PR. I wasn't aware that this was the proper way to encode these.